### PR TITLE
Updated module importer to follow new schema and spreadsheet

### PR DESCRIPTION
**Pull request by tommylau523**

### CODE REVIEW

**Comment by tommylau523** [comment]
* **#1872** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/1872?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @tommylau523 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/main/kotlin/com/terraformation/backend/accelerator/db/ModulesImporter.kt [hunk 1]
```diff
@@ -51,13 +64,27 @@ class ModulesImporter(
                 .insertInto(MODULES)
                 .set(ID, moduleId)
                 .set(NAME, name)
+                .set(OVERVIEW, overview)
+                .set(PREPARATION_MATERIALS, preparationMaterials)
+                .set(ADDITIONAL_RESOURCES, additionalResources)
+                .set(LIVE_SESSION_DESCRIPTION, liveSessionInfo)
+                .set(ONE_ON_ONE_SESSION_DESCRIPTION, oneOnOneInfo)
+                .set(WORKSHOP_DESCRIPTION, workshopInfo)
+                .set(PHASE_ID, CohortPhase.Phase1FeasibilityStudy)
```
Comment (sgrimm) [inline src/main/kotlin/com/terraformation/backend/accelerator/db/ModulesImporter.kt:73]: Will phase 1 be the only phase with modules? If not, I wonder if we should add a "phase" column to the spreadsheet right now so we don't forget to do it later.

**Review by sgrimm — APPROVED** [review]

**Comment by tommylau523** [comment]
### Merge activity

* **Apr 22, 7:41 PM EDT**: @tommylau523 merged this pull request with [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-server/1872).

File: src/main/kotlin/com/terraformation/backend/accelerator/db/ModulesImporter.kt (hunk 1 above)
Comment (tommylau523) [reply src/main/kotlin/com/terraformation/backend/accelerator/db/ModulesImporter.kt:73]: I believe there are talks of having Phase 0 modules. 
We should add the column to the spreadsheet but I want to confirm with the products folks first.

**Pull request merged**